src/tests/functional/plugin/cpu/subgraph_tests/src/reshape_inplace.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "functional_test_utils/ov_tensor_utils.hpp"
+#include "ngraph/runtime/aligned_buffer.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+#include "shared_test_classes/base/layer_test_utils.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+using namespace InferenceEngine;
+using namespace ov::test;
+namespace SubgraphTestsDefinitions {
+
```

Comment:
Could you please provide a minimal description with a scheme.
```
Const(data)     Parameter(shape)
          \     /
          Reshape
              |
          Output
```

Reply:
OK, will add!